src/DynamoCore/Core/Threading/UpdateRenderPackageAsyncTask.cs

```diff
@@ -213,6 +209,13 @@ private void GetRenderPackagesFromMirrorData(MirrorData mirrorData, ref List<str
                         package.PushTriangleVertex(d.X, d.Y, d.Z);
                         package.PushTriangleVertex(a.X, a.Y, a.Z);
 
+                        package.PushTriangleVertexUV(0,0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+                        package.PushTriangleVertexUV(0, 0);
+
```

Comment:
So, we're injecting a bunch of empty data?  Why not just flip a bit and say I've got no UV's.